src/main/java/com/terraformersmc/terrestria/init/TerrestriaDecoratedFeatures.java

```diff
@@ -180,10 +185,10 @@ public static void init() {
 
 		DENSEST_CYPRESS_TREES = decorateTree("densest_cypress_trees", 9, TerrestriaConfiguredFeatures.CYPRESS_TREE);
 
-		DENSER_RAINBOW_EUCALYPTUS_TREES = decorateTree("denser_rainbow_eucalyptus_trees", 5, TerrestriaConfiguredFeatures.RAINBOW_EUCALYPTUS_TREE);
+		DENSER_RAINBOW_EUCALYPTUS_TREES = decorateTree("denser_rainbow_eucalyptus_trees", 5, 3, TerrestriaConfiguredFeatures.RAINBOW_EUCALYPTUS_TREE);
 		DENSE_RUBBER_TREES = decorateTree("dense_rubber_trees", 3, TerrestriaConfiguredFeatures.RUBBER_TREE);
 
-		MEGA_CYPRESS_TREES = decorateTree("mega_cypress_trees", 2, TerrestriaConfiguredFeatures.MEGA_CYPRESS_TREE);
+		MEGA_CYPRESS_TREES = decorateTree("mega_cypress_trees", 2, 6, TerrestriaConfiguredFeatures.MEGA_CYPRESS_TREE);
```

Comment:
Where did you get the maxWaterDepth value (the added numbers) from?

Reply:
The tree feature configuration previously held the max water depth. It was removed from the `TerrestriaConfiguredFeatures` class.

Reply:
Ah, thanks for the answer! Didn't catch that they were moved over from there :)